internal/db/schema/migrations/oss/postgres/76/01_kms_oplog.up.sql

```diff
@@ -0,0 +1,31 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: BUSL-1.1
+
+begin;
+
+alter table oplog_entry
+  drop constraint iam_scope_fkey;
+alter table oplog_entry
+  drop constraint kms_data_key_version_fkey;
+
+
+drop trigger insert_oplog_entry on oplog_entry;
+drop function insert_oplog_entry();
+create function insert_oplog_entry() returns trigger
```

Comment:
I'm fine with adding a comment in the original.  When did we start adding
comments pointing back to the original?

Reply:
We've been adding them since the second release :) 

[internal/db/schema/migrations/oss/postgres/1/01_server_tags_migrations.up.sql ](https://github.com/hashicorp/boundary/blob/be8ba18dfe05da58f439b315d6949a292c5278eb/internal/db/schema/migrations/oss/postgres/1/01_server_tags_migrations.up.sql#L54-L55)

Reply:
seems sort of redundant, but I'm happy to add it.